package.json

```diff
@@ -27,7 +27,8 @@
     "detect-browser": "^1.7.0",
     "fetch-everywhere": "^1.0.5",
     "mongodb-extjson": "^1.0.5",
-    "query-string": "^4.3.4"
+    "query-string": "^4.3.4",
```

Comment:
i think you can put these into `devDependencies` since they're only necessary for the tests (for now)